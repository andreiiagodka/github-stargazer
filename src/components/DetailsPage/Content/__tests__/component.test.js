import { shallow } from 'enzyme'

import ContentComponent from '..'

describe('<ContentComponent />', () => {
  it('renders correctly', () => {
    const repository = {
      stargazers_count: 100,
      watchers_count: 200,
      forks_count: 300,
      languages: {
        JavaScript: 400
      }
    }

    const props = { repository: repository }
    const wrapper = shallow(<ContentComponent {...props} />)

    expect(wrapper).toMatchSnapshot()
  })
})
