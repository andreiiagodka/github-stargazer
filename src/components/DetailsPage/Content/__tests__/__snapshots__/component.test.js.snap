// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentComponent /> renders correctly 1`] = `
<Fragment>
  <Container
    fluid={false}
  >
    <Row
      noGutters={false}
    >
      <h3>
        Stats
      </h3>
    </Row>
    <Row
      noGutters={false}
    >
      <ul
        style={
          Object {
            "listStyleType": "none",
          }
        }
      >
        <li>
          Stars: 
          100
        </li>
        <li>
          Watchers: 
          200
        </li>
        <li>
          Forks: 
          300
        </li>
      </ul>
    </Row>
  </Container>
  <Container
    fluid={false}
  >
    <Row
      noGutters={false}
    >
      <h3>
        Languages
      </h3>
    </Row>
    <Row
      noGutters={false}
    >
      <ul
        style={
          Object {
            "listStyleType": "none",
          }
        }
      >
        <li
          key="JavaScript"
        >
          JavaScript
          : 
          400
        </li>
      </ul>
    </Row>
  </Container>
</Fragment>
`;
